endoreg_db/views/video/correction.py

```
@@ -0,0 +1,687 @@
+        
+        # Extract parameters
+        detection_method = request.data.get('detection_method', 'minicpm')
+        sample_interval = request.data.get('sample_interval', None)
+        
+        try:
+            # Initialize FrameCleaner with appropriate detection method
+            use_minicpm = detection_method in ['minicpm', 'hybrid']
+            frame_cleaner = FrameCleaner(use_minicpm=use_minicpm)
+            
+            # Get video path
+            video_path = video.raw_file.path if hasattr(video.raw_file, 'path') else str(video.raw_file)
```

Ruff (0.14.1)

Local variable `video_path` is assigned to but never used

Remove assignment to unused variable `video_path`

(F841)